tests/tests/block_schedule_tests.cpp

```diff
@@ -31,23 +31,73 @@ using namespace eos;
 using namespace chain;
 
 /*
+ * templated meta schedulers for fuzzing 
+ */
+template<typename NEXT>
```

Comment:
NEXT template argument looks like MACRO when.  Should be updated to conform to style standards.  "Next" for a template argument.